doc/source/installation.rst

```diff
@@ -44,6 +44,20 @@ if selinux is not enabled or is configured to be permissive.
 It is your reponsability to assure that soft dependencies of Ansible are
 available on your controller or host machines.
 
+.. warning::
+
+  It is highly recommended that you install molecule in a :std:doc:`virtual
+  environment <virtualenv:userguide>`. This will provide a modern copy of
+  `setuptools`_ which is mandatory in order for molecule to be installed
+  successfully and function correctly. If you cannot use a virtual environment
```

Comment:
@decentral1se apparently it's better to update "user" `setuptools` even when using virtualenv.